benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -975,25 +1133,240 @@ void GraphicsBenchmarkApp::DrawExtraInfo()
     ImGui::Columns(1);
 }
 
-void GraphicsBenchmarkApp::RecordCommandBuffer(PerFrame& frame, grfx::SwapchainPtr swapchain, uint32_t imageIndex)
+GraphicsBenchmarkApp::RenderPasses GraphicsBenchmarkApp::SwapchainRenderPasses(grfx::SwapchainPtr swapchain, uint32_t imageIndex)
+{
+    return RenderPasses{
+        swapchain->GetRenderPass(imageIndex, ppx::grfx::AttachmentLoadOp::ATTACHMENT_LOAD_OP_LOAD),
+        swapchain->GetRenderPass(imageIndex, ppx::grfx::AttachmentLoadOp::ATTACHMENT_LOAD_OP_CLEAR),
+        swapchain->GetRenderPass(imageIndex, ppx::grfx::AttachmentLoadOp::ATTACHMENT_LOAD_OP_DONT_CARE),
+        swapchain->GetRenderPass(imageIndex, ppx::grfx::AttachmentLoadOp::ATTACHMENT_LOAD_OP_LOAD),
+        swapchain->GetRenderPass(imageIndex, ppx::grfx::AttachmentLoadOp::ATTACHMENT_LOAD_OP_CLEAR),
+    };
+}
+
+ppx::Result GraphicsBenchmarkApp::CreateBlitContext(BlitContext& blit)
+{
+    // Descriptor set layout
+    {
+        // Descriptor set layout
+        grfx::DescriptorSetLayoutCreateInfo layoutCreateInfo = {};
+        layoutCreateInfo.bindings.push_back(grfx::DescriptorBinding(0, grfx::DESCRIPTOR_TYPE_SAMPLED_IMAGE));
+        layoutCreateInfo.bindings.push_back(grfx::DescriptorBinding(1, grfx::DESCRIPTOR_TYPE_SAMPLER));
+
+        PPX_CHECKED_CALL(GetDevice()->CreateDescriptorSetLayout(&layoutCreateInfo, &blit.descriptorSetLayout));
+    }
+
+    // Pipeline
+    {
+        std::vector<char> bytecode = LoadShader("basic/shaders", "FullScreenTriangle.vs");
```

Comment:
These two shaders could use `SetupShader()` maybe? You'd have to change the helper function so that the base folder is passed in though.

Reply:
Done.